src/core/net/dnssd_server.hpp

```diff
@@ -321,6 +322,16 @@ class Server : public InstanceLocator, private NonCopyable
     static constexpr uint16_t kRrTypeSrv   = ResourceRecord::kTypeSrv;
     static constexpr uint16_t kRrTypeAny   = ResourceRecord::kTypeAny;
 
+    // Recommended values for SOA record (RFC-8766 section 6.1).
```

Comment:
```suggestion
    // Recommended values for SOA record (RFC 8766 section 6.1).
```

Note that the style "RFC <number>" is the most common in OpenThread. "RFC-<number>" is only used in minority.